code/game/area/Space Station 13 areas.dm

```diff
@@ -2813,7 +2813,7 @@ var/list/the_station_areas = list (
 	name = "Keelin's private beach"
 	icon_state = "null"
 	luminosity = 1
-	lighting_use_dynamic = 0
+	dynamic_lighting = 0

```

Comment:
The only concern I have is how this will affect gateway maps. Please ensure that the gateway maps all function correctly. If they do not, it's probably because some still use "lighting_use_dynamic" in the area code.